src/main/java/org/javacord/bot/commands/InfoCommand.java

```diff
@@ -0,0 +1,28 @@
+package org.javacord.bot.commands;
+
+import de.btobastian.sdcf4j.Command;
+import de.btobastian.sdcf4j.CommandExecutor;
+import org.javacord.api.DiscordApi;
+import org.javacord.api.Javacord;
+import org.javacord.api.entity.channel.TextChannel;
+import org.javacord.api.entity.message.embed.EmbedBuilder;
+
+public class InfoCommand implements CommandExecutor {
+
+    @Command(aliases = "!info", async = true)
+    public void handleCommand(TextChannel channel) {
+        final DiscordApi api = channel.getApi();
+        EmbedBuilder embed = new EmbedBuilder()
+                .setTitle(api.getYourself().getName() + " - Official Javacord Bot")
+                .setThumbnail(api.getYourself().getAvatar())
+                .setDescription("The official bot for the Javacord Server.\n\n" +
+                        "Powered by Javacord and sdcf4j")
+                .addInlineField("GitHub", "https://github.com/Javacord/Javacord-Bot")
+                .addInlineField("Javacord Version", Javacord.VERSION)
+                .addInlineField("sdcf4j Version", "v1.0.10")
```

Comment:
I think it would be better to generate this information, or make it available programmatically from sdcf4j like for Javacord.
Like this, it will very easy get outdated and probably noone recognizes.
But we can also first have it hard-coded and I add some generation magic later if you like.